ground/lib/station.js

```diff
@@ -148,6 +139,14 @@ export class Station extends EventEmitter {
         resolve();
         return;
       }
+      
+      let dir = path.join(__dirname, '..', '..', '..', 'tools', 'rtlfm_demod.sh');
```

Comment:
unsure if it matters, but we might want to put `rtlfm_demod.sh` inside the npm module directory so it gets distributed properly. this relative pathing won't work if we try to `npm install` the ground station. have you tried this with `vg-console`/`vg-station`?